tests/test_radar_dataset.py

```diff
@@ -109,6 +110,24 @@ def test_radar_dataset(tmp_path):
     for graph, label in dataset:
         assert graph.num_edges() == min_neighbours**2
 
+    dataset = RadarDataset(
```

Comment:
Can you add a comment to the top explaining what this tests? In the future we will reorganize these tests, so it will be useful to have than info ready.

Reply:
The latest push should resolve this (and add more elaborate tests)